scanner/src/main/kotlin/LocalScanner.kt

```diff
@@ -100,15 +103,23 @@ abstract class LocalScanner(name: String, config: ScannerConfiguration) : Scanne
                     "Bootstrapping scanner '$scannerName' as required version $scannerVersion was not found in PATH."
                 }
 
-                bootstrap().also {
-                    val actualScannerVersion = getVersion(it)
-                    if (actualScannerVersion != scannerVersion) {
-                        throw IOException(
-                            "Bootstrapped scanner version $actualScannerVersion " +
-                                    "does not match expected version $scannerVersion."
-                        )
+                val (bootstrapDirectory, duration) = measureTimedValue {
+                    bootstrap().also {
+                        val actualScannerVersion = getVersion(it)
+                        if (actualScannerVersion != scannerVersion) {
+                            throw IOException(
+                                "Bootstrapped scanner version $actualScannerVersion " +
+                                        "does not match expected version $scannerVersion."
+                            )
+                        }
                     }
                 }
+
+                log.perf {
+                    "Bootstrapped scanner '$scannerName' version $scannerVersion in ${duration.inMilliseconds}ms."
```

Comment:
BTW, I hope that seeing how much time this takes convinces you that [removing the bootstrapping](https://github.com/oss-review-toolkit/ort/pull/3254) is the right thing to do 😉